src/Notification/Controller/ReadController.php

```diff
@@ -48,7 +48,12 @@ public function __construct(
     /**
      * @throws AccessDeniedException|NotFoundException|UserNotFoundException
      */
-    #[Route('/notifications/{id}', name: 'pimcore_studio_api_read_notification', methods: ['POST'])]
+    #[Route(
+        '/notifications/{id}',
+        name: 'pimcore_studio_api_read_notification',
+        requirements: ['id' => '\d+'],
```

Comment:
This is new no? Should we introduce that for every path id?

Reply:
We already have it e.g here: https://github.com/pimcore/studio-backend-bundle/blob/1.x/src/Asset/Controller/GetController.php#L52

it just ensures that route is not overwriting sub - routes. I would say we should use it only when necessary